.github/DEVELOPMENT.md

```diff
@@ -8,11 +8,10 @@ This page contains steps to build and run the .NET MAUI repository from source.
 
 In most cases, when you have Visual Studio installed with the .NET workloads checked, these steps are not required.
 
-1. Install the latest .NET 6:  
-   <!--- [Win (x64)](https://aka.ms/dotnet/6.0.2xx/daily/dotnet-sdk-win-x64.exe)   -->
+1. Install the latest .NET 6:
    - [Install the latest Public Preview of Visual Studio](https://docs.microsoft.com/en-us/dotnet/maui/get-started/installation/)
-   - [macOS (x64)](https://aka.ms/dotnet/6.0.2xx/daily/dotnet-sdk-osx-x64.pkg)  
-   - [macOS (arm64)](https://aka.ms/dotnet/6.0.2xx/daily/dotnet-sdk-osx-arm64.pkg)
+   - [macOS (x64)](https://aka.ms/dotnet/6.0.3xx/daily/dotnet-sdk-osx-x64.pkg)  
+   - [macOS (arm64)](https://aka.ms/dotnet/6.0.3xx/daily/dotnet-sdk-osx-arm64.pkg)
```

Comment:
Not sure if the IDEs are ready for the 300 series... I think it might, but I was having workload issues last week. Need to test again. Have you had success?

Reply:
Note this change is specifically for MacOS.

I'm working on 6.0.3xx on MacOS (from command line) with success. Unfortunately, can't get any to work in VS4Mac as I get a message saying MAUI isn't yet supported (Version 17.1 Preview (17.1 build 2)).

Reply:
<img width="751" alt="Pasted Graphic" src="https://user-images.githubusercontent.com/14852843/156100481-ad1b6e7c-e247-4c07-9b0e-10bb57e50f7e.png">